src/test/scala/uk/gov/hmrc/test/ui/pages/section1/DeclarationTypePage.scala

```diff
@@ -55,6 +52,13 @@ object DeclarationTypePage extends BasePage {
     store(AdditionalDeclarationType -> Detail(additionalDeclarationType))
   }
 
-  def isArrivedDeclaration: Boolean = detail(AdditionalDeclarationType).startsWith("Arrived -")
-  def isPrelodgedDeclaration: Boolean = detail(AdditionalDeclarationType).startsWith("Pre-lodged -")
+  def isArrivedDeclaration: Boolean = {
+    val adt = detail(AdditionalDeclarationType)
+    adt.startsWith("Arrived -") || adt.startsWith("EIDR -")
+  }
+
+  def isPrelodgedDeclaration: Boolean = {
+    val adt = detail(AdditionalDeclarationType)
+    adt.startsWith("Pre-lodged -") || adt.startsWith("Simplified -")
```

Comment:
Please, leave it as I did, because "Simplified" has a different meaning than "Pre-lodged". You can add another method `isSimplified` and then, when you need the two conditions, you can write something like `isPrelodged || isSimplified`.